cmd/reconciler-manager/main.go

```diff
@@ -70,8 +71,8 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	flag.Parse()
 
+	log.Setup()
```

Comment:
It looks like the `init` function above hard codes the `v` flag to `1`. I expect we need to remove that so that we use the actual verbosity flag that is passed in to the binary

Reply:
good catch. I guess I'll just set it to -v=1 in the manifest

Reply:
IIRC it doesn't need to be set in the manifest any more, it gets set in code now with the new logLevel api (getting a default value from code)

Reply:
nope. that's just for reconciler containers.

Reply:
oh I guess this is reconciler-manager, nevermind

Reply:
done